model.go

```diff
@@ -82,16 +83,21 @@ type (
 	// KafkaMocks is keyed by Topic
 	KafkaMocks map[ExpectKafka]MocksArray
 
+	// GRPCMocks is keyed by service/method
+	GRPCMocks map[ExpectGRPC]MocksArray
+
 	// AMQPMocks is keyed by Queue name
 	AMQPMocks map[ExpectAMQP]MocksArray
 
 	// MockRepo stores a repository of Mocks
 	MockRepo struct {
 		HTTPMocks  HTTPMocks
+		GRPCMocks  GRPCMocks
 		KafkaMocks KafkaMocks
 		AMQPMocks  AMQPMocks
 		Templates  MocksArray
-		Behaviors  map[string]*Mock
+
+		Behaviors *orderedmap.OrderedMap
```

Comment:
introduced `ordered map` to preserve the order of insertion into the map, so that we can loop through the conditions matching across mocks while respect their order.